configs/openSUSE/dbus-services.toml

```diff
@@ -285,11 +285,21 @@ nodigests = [
 [[FileDigestGroup]]
 package = "bluez"
 type = "dbus"
-note = "imported from rpmlint1 DBUSServices.WhiteList"
-bug = "bsc#768062"
-nodigests = [
-    "/etc/dbus-1/system.d/bluetooth.conf",
-    "/usr/share/dbus-1/system-services/org.bluez.service",
+note = "Provides bluetooth stack access via D-Bus"
+bugs = ["bsc#768062", "bsc#1199207"]
+digests = [
+    {
+        path = "/usr/share/dbus-1/system.d/bluetooth.conf",
+        algorithm = "sha256",
+        digester = "xml",
+        hash = "529151093a482e8223738f86bec3dac4c23733c38fba1f318cc6130b5acaebfe"
```

Comment:
The current hash in openSUSE:Factory/bluez and also Base:System/bluez is `c211c4412f39dec5dd93c3da9a8b4bbeec37825fdb74d5fd8efd23573f11f1ed`

Reply:
It looks like you digested the plain file, this digest is for the XML filtered file, please use `tools/get_whitelisting_digest.py` from the repository to compare digests.

Reply:
Yes, I digested the plain file. Sorry for the noise.
`529151093a482e8223738f86bec3dac4c23733c38fba1f318cc6130b5acaebfe` is correct.